controllers/front/webhook.php

```diff
@@ -65,8 +63,10 @@ public function postProcess()
             exit;
         }
 
-        // Confirm the order with MoneyBadger
-        $this->confirmInvoice($cryptoReference);
+        // Confirm the order with MoneyBadger if it's paid
+        if ($orderStatus === $PS_ORDER_STATUS_PAID) {
```

Comment:
Need to check this, we were getting 500s trying to confirm timed-out invoices.

Reply:
I suspect you want to wrap the confirm in a try/catch and if you get an error set the orderstatus to error